boa/core/test.py

```diff
@@ -64,7 +64,8 @@ def get_metadata(yml, config, is_pyproject_recipe=False):
 
                 d = toml.load(fi)["tool"]["boa"]
         else:
-            d = ruamel.yaml.safe_load(fi)
+            loader = YAML(typ="safe")
```

Comment:
The equivalent of `safe_load(…)` is documented to be `YAML(typ="safe", pure=True).load(…)`. The `pure=True` argument is noted in the deprecation warning and should likely be used here.